pkg/profiler/cpu/cpu.go

```diff
@@ -504,7 +504,7 @@ func (p *CPU) Run(ctx context.Context) error {
 	if err != nil {
 		return fmt.Errorf("failed to init perf buffer: %w", err)
 	}
-	perfBuf.Start()
+	perfBuf.Poll(250)
```

Comment:
nit/question: Is `250` the default value? So there's no behavioral change, right?

Reply:
Good question! The default value is currently 300. Perhaps we can lower it to 250 and see what the overhead is in production. We can fine-tune this value to find the right trade-off for us regarding CPU usage and the ability to catch short-running processes